tetris.py

```diff
@@ -27,9 +27,12 @@ def run_gui():
     root = tkinter.Tk()
     root.resizable(False, False)
 
-    topbar = tkinter.Label(root)
-    topbar['bg'] = D_GREY
+    topbar = tkinter.Frame(root, bg=D_GREY, height=square_size*2, relief = 'ridge', borderwidth=1)
     topbar.grid(row=0, column=0, sticky='we')
+
+    topbar_next_block = tkinter.Label(topbar)
```

Comment:
I guess this should be a `Canvas`, if you want to actually display a block, rather than just the shape name.

Reply:
Adjusted